### DISCUSSION TITLE
Authentication access denied

### DISCUSSION THREAD

**Discussion by Shimmybaks** [community/community#61751] [New to GitHub]
Hi there

I'm having issues to push my work ?remote: Permission to Shimmybaks/printf.git denied to Shimmybaks.
fatal: unable to access 'https://github.com/Shimmybaks/printf.git/': The requested URL returned error: 403
please assist

**Comment by Manishak798** [comment]
First, you need to check if you have permission to push to that repo.
The error message “the requested URL returned error: 403” indicates that you are forbidden from accessing the requested URL. This is most likely due to an issue with the personal access token you are using to push to GitHub.
Hope it helps!!
